coderd/database/queries.sql.go

```diff
@@ -490,7 +490,7 @@ type GetAuditLogsOffsetRow struct {
 	UserEmail        sql.NullString  `db:"user_email" json:"user_email"`
 	UserCreatedAt    sql.NullTime    `db:"user_created_at" json:"user_created_at"`
 	UserStatus       NullUserStatus  `db:"user_status" json:"user_status"`
-	UserRoles        []string        `db:"user_roles" json:"user_roles"`
+	UserRoles        pq.StringArray  `db:"user_roles" json:"user_roles"`
```

Comment:
Not a fan of this change, can we disable this functionality?

Reply:
this is a generated file so maybe sqlc has changed this

Reply:
Yeah I know, but if we can disable it with e.g. a flag then we probably should

Reply:
It's weird that it changed, but I don't think it would be a problem. Most of the time slice types are interchangeable: https://go.dev/play/p/O765z2BGVJ3

Reply:
@deansheather what's the issue with `pq.StringArray`?

Reply:
TBH I thought it would be some silly struct wrapper type, if it's just a type alias then I guess it's fine